app/jobs/appointment_reminders/send_reminder_job.rb

```diff
@@ -0,0 +1,73 @@
+class AppointmentReminders::SendReminderJob
+  include Rails.application.routes.url_helpers
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :high
+
+  # i think this should be based on country instead
+  DEFAULT_LOCALE = :en
+
+  def perform(reminder_id)
+    reminder = AppointmentReminder.includes(:appointment, :patient).find(reminder_id)
+    send_message(reminder)
+  end
+
+  private
+
+  def send_message(reminder)
+    notification_service = NotificationService.new
+    # i think this logic should be handled by the notification service
```

Comment:
I'm planning to move the error handling into the NotificationService. I've looked at how it's used around the app and it seems safe to move error reporting to the class that would be generating the error.